lock-impl/src/main/java/com/palantir/lock/impl/LockServiceImpl.java

```diff
@@ -1187,17 +1184,12 @@ public void logCurrentState() {
         StringBuilder logString = getGeneralLockStats();
         log.info("Current State: {}", SafeArg.of("state", logString.toString()));
 
-        try {
-            logAllHeldAndOutstandingLocks();
-        } catch (IOException e) {
-            log.error("Can't dump state to Yaml", e);
-            throw new IllegalStateException(e);
-        }
+        logAllHeldAndOutstandingLocks();
```

Comment:
we might want to just combine these all into one log line statement. I'm a little concerned with how much `getGeneralLockStats()` allocates if we're logging (frequently?) when lock services are already in a potentially degraded performance state